### ISSUE TITLE
sqlite3.OperationalError: no such column: disqualified

### ISSUE THREAD

**Issue by jxtn-rpg** [ReneSmeekes/storj_earnings#15]
earnings.py v9.3.1
storagenode v1.12.3

```
Traceback (most recent call last):
  File "earnings.py", line 283, in <module>
    for data in con.execute(query):
sqlite3.OperationalError: no such column: disqualified
```

**Comment by ReneSmeekes** [comment]
Some changes to the database structure have been made with storagenode version 1.12. My nodes are on docker, so I'm still waiting for this update to roll out. I'll implement fixes/adjustments as soon as I'm on the new version.

**Comment by ReneSmeekes** [comment]
Looks like my nodes were just updated. I'll try to get the fix out of the door asap!

**Comment by ReneSmeekes** [comment]
Fixed now in v9.4.0!
